pkg/builds/cosa_v1.go

```diff
@@ -92,6 +92,7 @@ type BuildArtifacts struct {
 	Exoscale            *Artifact `json:"exoscale,omitempty"`
 	ExtensionsContainer *Artifact `json:"extensions-container,omitempty"`
 	Gcp                 *Artifact `json:"gcp,omitempty"`
+	Hyperv              *Artifact `json:"hyperv,omitempty"`
```

Comment:
This is a generated file.  You should edit `schema/v1.json` and run `make schema`.